.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       fail-fast: true
       matrix:
         python: ["3.7", "3.10"]
-        os: [ubuntu-latest] # TODO: macos-latest, windows-latest
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
Opened issue #9 for macOS arm support and #10 for more generic manylinux support